pkg/controllermanager/controller/controllerregistration/controllerregistration_control.go

```diff
@@ -278,3 +293,13 @@ func (c *defaultControllerRegistrationControl) delete(controllerRegistration *ga
 	})
 	return err
 }
+
+func convertObjToMap(in interface{}) (map[string]interface{}, error) {
+	var out map[string]interface{}
+	data, err := json.Marshal(in)
+	if err != nil {
+		return nil, err
+	}
+	json.Unmarshal(data, &out)
```

Comment:
This also returns an error - I know, it should not, but for the sake of tougher golint checks we should also cover this error here.